ui/apps/platform/src/Containers/Policies/Wizard/Step1/PolicyMetadataFormSection.tsx

```diff
@@ -1,160 +1,136 @@
-    FormHelperText,
-    HelperText,
-    HelperTextItem,
-} from '@patternfly/react-core';
-import { Field, useFormikContext } from 'formik';
+import { Flex, TextInput, Radio, TextArea, Form } from '@patternfly/react-core';
+import { FormikContextType, useFormikContext } from 'formik';
+
+import FormLabelGroup from 'Components/PatternFly/FormLabelGroup';
+import { ClientPolicy } from 'types/policy.proto';
 
 import PolicyCategoriesSelectField from './PolicyCategoriesSelectField';
 
 function PolicyMetadataFormSection(): ReactElement {
-    const { handleChange } = useFormikContext();
+    const {
+        errors,
+        handleChange,
+        handleBlur,
+        setFieldValue,
+        touched,
+        values,
+    }: FormikContextType<ClientPolicy> = useFormikContext();
 
-    function onChange(_value, event) {
-        handleChange(event);
-    }
     return (
         <Form>
-            <Field name="name">
-                {({ field }) => (
-                    <FormGroup fieldId="policy-name" label="Name" isRequired>
-                        <FormHelperText>
-                            <HelperText>
-                                <HelperTextItem>
-                                    Provide a descriptive and unique policy name
-                                </HelperTextItem>
-                            </HelperText>
-                        </FormHelperText>
-                        <TextInput
-                            id={field.name}
-                            name={field.name}
-                            value={field.value}
-                            onChange={(event, _value) => onChange(_value, event)}
-                            isRequired
-                        />
-                    </FormGroup>
-                )}
-            </Field>
-            <FormGroup fieldId="policy-severity" label="Severity" isRequired>
-                <FormHelperText>
-                    <HelperText>
-                        <HelperTextItem>Select a severity level for this policy</HelperTextItem>
-                    </HelperText>
-                </FormHelperText>
+            <FormLabelGroup
+                label="Name"
+                isRequired
+                fieldId="name"
+                errors={errors}
+                touched={touched}
+                helperText={'Provide a descriptive and unique policy name'}
+            >
+                <TextInput
+                    isRequired
+                    type="text"
+                    id="name"
+                    name="name"
+                    value={values.name}
+                    validated={errors?.name && touched?.name ? 'error' : 'default'}
+                    onChange={(event) => handleChange(event)}
+                    onBlur={handleBlur}
+                />
+            </FormLabelGroup>
+            <FormLabelGroup
+                label="Severity"
+                fieldId="severity"
+                errors={errors}
+                isRequired
```

Comment:
Very small: can we move `isRequired` so `FormLabelGroup` element has consistent order of props?

Reply:
I still struggle with this one since I haven't really noticed a consistency across all of our components in how we order props, but I guess per file is give me.